libycp/src/include/ycp/YCPBoolean.h

```diff
@@ -98,6 +99,7 @@ class YCPBoolean : public YCPValue
     static YCPBoolean* falseboolean;
     
 public:
+    YCPBoolean(const boost::logic::tribool &tb) : YCPBoolean((bool)tb) {}
```

Comment:
where is this tribool used? So far I see in core only usage of some trivial string algorithm from boost.

Reply:
https://bugzilla.suse.com/show_bug.cgi?id=1128364

so the fixes is either to explicitly cast it there in every usage or provide this cast here.

Reply:
@lslezak can you please check it? I am not sure if there is not some consequences for tribool to boolean conversion in some places in pkg-bindings.
also @mvidner if you can check it.

Reply:
I am making a better fix in yast-pkg-bindings.